manifests/course/practitioner.pp

```diff
@@ -22,4 +22,6 @@
   class { 'classroom::facts':
     coursename => 'practitioner',
   }
+
+  Yumrepo <| |> -> Package <| |>
```

Comment:
Actually I think this line might not be needed anymore.  I had originally done it before moving some of the package installation to build time.